docs/development/release.rst

```diff
@@ -39,50 +60,75 @@ On the day of the release candidate:
    * In the ``download/install`` folder, copy a previous environment file file as ``gammapy-1.0rc1-environment.yml``.
    * Adapt the dependency conda env name and versions as required in this file.
 
-#. Update the ``CITATION.cff`` date and version by running the ``dev/prepare-release.py`` script.
-#. Locally create a new release candidate tag on the v1.0.x, like ``v1.0rc1`` for Gammapy and push. For details see the
-   `Astropy release candidate instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#releasing-the-first-feature-release-candidate>`__.
-#. Once the tag is pushed the docs build and upload to `PyPi <https://pypi.org/>`__ should be triggered automatically.
-#. Once the docs build has succeeded find the ``tutorials_jupyter.zip`` file for the release candidate
-   in the `gammapy-docs repo <https://github.com/gammapy/gammapy-docs>`__ and adapt the ``download/index.json`` to point to it.
-#. Update the entry for the release candidate in the `Gammapy release calendar <https://github.com/gammapy/gammapy/wiki/Release-Calendar>`__.
-#. Create a testing page like `Gammapy v1.0rc testing <https://github.com/gammapy/gammapy/wiki/Gammapy-v1.0rc-testing>`__.
+#. Switch to the correct branch and update the ``CITATION.cff`` date and version by running the
+   ``dev/prepare-release.py`` script::
+
+    git checkout v1.0.x
+    python ./dev/prepare-release.py --release v1.0rc1
+
+#. Commit and push the branch back to GitHub::
+
+    git push upstream v1.0.x
+
+#. Locally create a new release candidate tag on the ``v1.0.x``, like ``v1.0rc1`` for Gammapy and push::
+
+    git tag -s v1.0rc1 -m "Tagging v1.0rc1"
+    git push upstream v1.0.x
+
+#. Once the tag is pushed, the docs build and upload to `PyPi <https://pypi.org/>`__ should be triggered automatically.
+#. Check the ``Actions`` on `gammapy repo <https://github.com/gammapy/gammapy>`__ to check that the necessary
+   actions have started.
+#. Once the docs build is successful find the ``tutorials_jupyter.zip`` file for the release candidate in the
+   `gammapy-docs repo <https://github.com/gammapy/gammapy-docs>`__ and adapt the ``download/index.json`` to point to it.
+#. Update the entry for the release candidate in the
+   `Gammapy release calendar <https://github.com/gammapy/gammapy/wiki/Release-Calendar>`__.
+#. Create a testing page like
+   `Gammapy v1.0rc testing <https://github.com/gammapy/gammapy/wiki/Gammapy-v1.0rc-testing>`__.
 #. Advertise the release candidate and motivate developers and users to report test fails and bugs and list them
    on the page created before.
 
 
 Releasing the final version of the major release
 ------------------------------------------------
 
-#. Create a new release tag in the `gammapy-data repo <https://github.com/gammapy/gammapy-data>`__, like ``v1.0`` or ``v1.1``.
+#. Create a new tag in the `gammapy-data repo <https://github.com/gammapy/gammapy-data>`__, like ``v1.0``
+   or ``v1.1``.
```

Comment:
Should we add here the detailed instructions?

Reply:
It is simply adding a new tag here: https://github.com/gammapy/gammapy-data/tags :)